Source/Krypton Components/Krypton.Toolkit/General/Definitions.cs

```diff
@@ -3087,4 +3087,29 @@ public enum KryptonUseRTLLayout
 
     #endregion
 
+    #region Struct HELPINFO
```

Comment:
These should go into the same place as the other `PInvoke structs`

Reply:
Fixed locations